Automattic/woocommerce-payments
# Make success notice undismissable on overview page.

Fixes #1975 

#### Changes proposed in this Pull Request
For now, I have made the success notice which shows up after successful KYC undismissable. This will automatically go away once the route/url is changed. Also, that seems to be consistent with other notices, let me know if that is fine, else we can make it dismissable as well.









#### Testing instructions
- switch to `fix/1975-make-notice-indismissible`
- complete the onboarding step via connect screen.
- you'll see a success notice after redirection which is not dismissable anymore.

#### After fix:
![image](https://user-images.githubusercontent.com/15019298/120331949-e80aa380-c30b-11eb-96f0-3730a31aca84.png)


-------------------
